README.rst

```diff
@@ -18,6 +18,47 @@ This is the Catalog micro-frontend application, currently under development.
 These are public-facing pages intended for unauthenticated users.
 The goal is to replace legacy views in ``edx-platform`` with modern, React and Paragon-based implementations.
 
+Installing
+**********
+
+As of the `Open edX Ulmo release <https://docs.openedx.org/en/latest/community/release_notes/ulmo.html>`_,
```

Comment:
@brian-smith-tcril I made a placeholder page for the Catalog release note. Could you take a look (https://github.com/openedx/docs.openedx.org/pull/1327) and once it merges we can link to that page from here.

Reply:
that's exactly what I had in mind!